logservice/schemastore/persist_storage_ddl_handlers.go

```diff
@@ -634,10 +635,23 @@ func buildPersistedDDLEventForRenameTable(args buildPersistedDDLEventFuncArgs) P
 		// And because SchemaStore is the source of truth inside cdc,
 		// we can use event.ExtraSchemaID(even it is wrong) to update the internal state of the cdc.
 		// But event.Query will be emit to downstream(out of cdc), we must make it correct.
+		//
+		// InvolvingSchemaInfo returns the schema info involved in the job.
+		// The value should be stored in lower case.
+		oldSchemaName := args.job.InvolvingSchemaInfo[0].Database
+		oldTableName := args.job.InvolvingSchemaInfo[0].Table
+		stmt, err := parser.New().ParseOneStmt(args.job.Query, "", "")
+		if err != nil {
+			log.Error("parse statement failed for build persisted DDL event", zap.Any("DDL", args.job.Query), zap.Error(err))
```

Comment:
Don't print error log when parse fail, just print warning, and continue to construct the query.